docs/en_US/deployment/on-gcp-gke.md

```diff
@@ -17,22 +17,30 @@ Before you begin, you must have the following:
 
 - To install EMQX Operator, please refer to [Install EMQX Operator](../getting-started/getting-started.md)
 
-  ::: warning
-  For installing `cert-manager` on Google Kubernetes Engine, see the official documentation:
-
-  - [GKE Autopilot](https://cert-manager.io/docs/installation/compatibility/#gke-autopilot)
-  - [Private GKE Cluster](https://cert-manager.io/docs/installation/compatibility/#gke)
 
-  Remember to install CRDs when running `helm` with the `--set installCRDs=true` flag.
-
-  More information can be found at [cert-manager](https://cert-manager.io).
-  :::
+## Install cert-manager
```

Comment:
Add some `warning` remind the user that `--set global.leaderElection.namespace=cert-manager` is required.